src/CollectdWinService/config/WriteHttpPluginConfig.cs

```diff
@@ -54,6 +54,27 @@ public int MaxIdleTime
                 set { base["MaxIdleTime"] = value; }
             }
 
+            [ConfigurationProperty("UserName", IsRequired = false)]
+            public string UserName
+            {
+                get { return (string)base["UserName"]; }
+                set { base["UserName"] = value; }
+            }
+
+            [ConfigurationProperty("Password", IsRequired = false)]
+            public string Password
+            {
+                get { return (string)base["Password"]; }
+                set { base["Password"] = value; }
+            }
+
+            [ConfigurationProperty("UseSafeNames", IsRequired = false)]

```

Comment:
Thx, no problem.  In WriteHttp.config under node, or WindowsPerformanceCounter.config?  Since I can see multi-targets, some perhaps using the replacement and others not, if it's in WindowsPerformanceCounterPlugin.cs then it will need to know the target scope (http/node) to prevent global replace, but WindowsPerformanceCounterPlugin.cs is not target aware and I'm not sure it's supposed to be.